zebra/zebra_dplane.c

```diff
@@ -4132,19 +4149,38 @@ dplane_route_update_internal(struct route_node *rn,
 	enum zebra_dplane_result result = ZEBRA_DPLANE_REQUEST_FAILURE;
 	int ret = EINVAL;
 	struct zebra_dplane_ctx *ctx = NULL;
-
+	struct nexthop *nexthop, *old_nexthop;
 	/* Obtain context block */
 	ctx = dplane_ctx_alloc();
+	if (IS_ZEBRA_DEBUG_KERNEL) {
```

Comment:
How does this debug help the end developer debug this.

Reply:
Just for me to observe flags' change. Should I delete them?